dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -233,12 +233,20 @@ def test_add_profilers(self, *mocks):
         self.assertEqual(
             "<class 'pandas.core.frame.DataFrame'>", merged_profile.file_type
         )
+        self.assertTrue(merged_profile.options.row_statistics.null_count.is_enabled)
         self.assertEqual(2, merged_profile.row_has_null_count)
         self.assertEqual(2, merged_profile.row_is_null_count)
         self.assertEqual(7, merged_profile.total_samples)
         self.assertEqual(5, len(merged_profile.hashed_row_object))
         self.assertDictEqual({"row_stats": 2}, merged_profile.times)
 
+        # test success when null_count option is disabled
```

Comment:
should I create a new test function instead of bunching this into test_add_profilers?

Reply:
yes, that's a fair call -- just be very descriptive with what the test is doing in the test name such as `def test_add_profiles_null_count_not_enabled()`